Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -117,13 +119,17 @@ protected bool TestObjectExists(StorageService service, string bucket, string ob
     /// <example>
     ///   <para>Upload a local log file to GCS.</para>
     ///   <para><code>PS C:\> New-GcsObject -Bucket "widget-co-logs" -ObjectName "log-000.txt" `</code></para>
-    ///   <para><code>    -File "C:\logs\log-000.txt"</code></para>
+    ///   <para><code>    -Path "C:\logs\log-000.txt"</code></para>
```

Comment:
Shouldn't this still be -File? Reading the comment above it sounds like I can either use -Contents, -File, or -Path. If -File was removed, we should update the comment on line 108.